galaxykubeman/templates/rbac.yaml

```diff
@@ -0,0 +1,30 @@
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: {{ .Release.Namespace }}-rbac
+  namespace: {{ .Release.Namespace }}
+rules:
+  - apiGroups: ["", "extensions", "apps"]
+    resources: ["*"]
+    verbs: ["*"]
+  - apiGroups: ["batch"]
+    resources:
+      - jobs
+      - cronjobs
+    verbs: ["*"]
```

Comment:
Just wanted to point out that the rules on the [proof-of-concept doc](https://broadworkbench.atlassian.net/wiki/spaces/IA/pages/454033444/2020-05-14+Restricting+Cloud+Man+s+Permissions) had the values above, as an example that worked for the purpose of prototyping. If it's possible to make the rules stricter, we should do that. I'm not sure what the minimum RBAC needed for Galaxy app installation is.

Reply:
I spoke to @almahmoud about this and he said that at first glance, these rules look fine. If we decide to make them stricter then I can update this PR!